desc/vmec.py

```diff
@@ -552,19 +570,15 @@ def save(cls, eq, path, surfs=128, verbose=1):  # noqa: C901 - FIXME - simplify
             ["<|B|^2>", "<J*B>", "sqrt(g)", "J^theta*sqrt(g)", "J^zeta", "D_Mercier"],
             grid=grid_full,
         )
-        data_half = eq.compute(["I", "G"], grid=grid_half)
+        data_half = eq.compute(["I", "G", "<|B|^2>", "V_r(r)"], grid=grid_half)
```

Comment:
We also compute the profiles on the half grid around line 195, can that be combined with this?

There's also another half and full grid data computed on line ~833 that could be combined here